src/components/Grid/Grid.scss

```diff
@@ -17,12 +17,16 @@ $border-color: #dbdbdb;
   
   &.block {
     flex-direction: column;
-    min-width: 180px;
+    min-width: 17.3rem;
```

Comment:
Can you explain why a min-width value is required.

Reply:
@weegrom as per our discussion i had removed min-width from grid